### PR TITLE
Salesforce approvals + sync Salesforce to database

### PR DESCRIPTION
**Pull request by graceliching**
### Which issues does this fix?
Fixes #12345

### PR submission checklist
- [ ] Checked all content for grammatical errors (using Grammarly, for example)
- [ ] Added new files to `docs/vuepress/sidebar.js`
- [ ] Check for broken links
```bash
npm run check
```

### CODE REVIEW

File: docs/connectors/salesforce/approval-processes.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+- A queue specified by the administrator.
+- A user listed in the Manager standard field on the submitter’s user detail page.
+- A user listed in a custom hierarchy field on the submitter’s user detail page.
+- Any combination of users and related users (users listed in a standard or custom field on the submitted record) specified by the administrator.
+
+
+## Submit record for approval
+This action does the equivalent of clicking on the 'Submit for approval' button on a Salesforce record. Doing this via a Workato recipe automates this manual click.
+
+This action requires the following inputs:
```
Comment (eeshansim) [inline docs/connectors/salesforce/approval-processes.md:51]: Better to include an image of the action

File: docs/connectors/salesforce/approval-processes.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+*An Invoice record submitted for approval*
+
+## Approve record in an approval process
+This action approves the record and all approval steps defined in the approval process are triggered. This could include the record getting locked, field updates, an outbound message or an email alert being sent out.
+
+If the approval steps automatically sends the record through the 'Final approval actions', these steps will also be performed.
+
+The 'Approve record' and 'Reject record' actions should be used in the logic flow of your recipe where conditions are defined for approval and rejection accordingly.
+
+This action requires the following inputs:
```
Comment (eeshansim) [inline docs/connectors/salesforce/approval-processes.md:73]: Better to include an image of the action

File: docs/connectors/salesforce/approval-processes.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+*An approved record*
+
+## Reject record in an approval process
+This action rejects the record and all rejection steps defined in the approval process are triggered. This could include the record getting locked, field updates, an outbound message or an email alert being sent out.
+
+If the rejection steps automatically sends the record through the 'Final rejection actions', these steps will also be performed.
+
+The 'Approve record' and 'Reject record' actions should be used in the logic flow of your recipe where conditions are defined for approval and rejection accordingly.
+
+This action requires the following inputs:
```
Comment (eeshansim) [inline docs/connectors/salesforce/approval-processes.md:92]: Better to include an image of the action

File: docs/connectors/salesforce/database-syncs.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+Integrations between Salesforce and databases are useful when there is a need to optimize organizational processes: Using an integration product to do so greatly reduces the cost of manually addressing inconsistencies of aligning data in different systems.
+
+## Get object schema action
+
+The 'Get object schema' action fetch schema information of a Salesforce object. This action can be used on both standard and custom objects. Use this action when you need to replicate a Salesforce object's schema to your data warehouse.
+
+This action requires the user to select the object to fetch object schema from. The example snippet below shows the output when the 'Opportunity' object is selected.
+
+```
+{
```
[the remark this answers is not in the document]
Comment (graceliching) [reply docs/connectors/salesforce/database-syncs.md:18]: The test case example only has the output from the object fields, whereas the action returns other metadata fields as well. Will this be confusing to users?

File: docs/connectors/salesforce/database-syncs.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+
+## Detect new or updated custom data
+
+This boolean field is available in Salesforce triggers for all single-record and batch New, New/Updated record triggers as well as Scheduled record search using SOQL. Use this field only if trying to replicate object records to a database.
+
+When 'Yes' is selected, the trigger output will include data of any newly added or modified custom fields. It is then used in conjunction with a replicate data action in the corresponding database. This is currently supported on the Snowflake adapter.
+
+When new/updated data is detected from the data source, Workato first inspects the new object definition against the existing Snowflake table schema. Any mismatches will be reconciled by updating the table definition in Snowflake. Following that, the data is then deduplicated and inserted/updated in the Snowflake table.
+
+View this [recipe](https://www.workato.com/recipes/1145888-saleforce-contacts-snowflake-batch-replicate) for an example of how this is done.
```
[the remark this answers is not in the document]
Comment (graceliching) [reply docs/connectors/salesforce/database-syncs.md:124]: Do you think these recipes we showcase in docs should come from 1 account/a Workato solutions demo account?

[the remark this answers is not in the document]
Comment (graceliching) [reply docs/connectors/salesforce/database-syncs.md:124]: or do you have rights to make your recipe a 'Featured' recipe? I used to but it doesn't work for me now

[the remark this answers is not in the document]
Comment (eeshansim) [reply docs/connectors/salesforce/database-syncs.md]: Copied into solution demo account: https://www.workato.com/recipes/1169161?st=39eedb

File: docs/connectors/salesforce/database-syncs.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (eeshansim) [reply docs/connectors/salesforce/database-syncs.md:18]: The incoming PR will remove all excess values.

**Pull request merged**